src/configure/resources/constants.ts

```diff
@@ -196,4 +196,8 @@ export const ingressManifest: string = "ingress";
 export const azurePipeline: string = "Azure-pipeline";
 export const githubWorkflow: string = "Github-workflow";
 export const clientPropertyKey: string = "ms.client.vscode";
-export const inputModeProperty: string = "inputMode";
\ No newline at end of file
+export const inputModeProperty: string = "inputMode";
+
+export const ExceptionType = {
+    UnauthorizedRequestException: 'UNAUTHORIZEDREQUESTEXCEPTION'
```

Comment:
do you want to use underscores to make it more readable

Reply:
The right side value is the exception came from the API response. To avoid case ignore comparison, I have converted into upper case.